packages/google_sign_in/google_sign_in_web/lib/src/js_interop/gapiauth2.dart

```diff
@@ -12,7 +12,9 @@
 
 // https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/gapi.auth2
 
-// ignore_for_file: public_member_api_docs, unused_element, non_constant_identifier_names, sort_constructors_first, always_specify_types, strict_raw_type
+// ignore_for_file: public_member_api_docs, non_constant_identifier_names,
+// * non_constant_identifier_names required to be able to use the same parameter
+//   names as the underlying library.
```

Comment:
Remember to document here why are you adding the `public_member_api_docs` to this file. The explanation can be along the lines of "originally undocumented because the file was autogenerated." or similar.

Reply:
Got it. Done.